ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -874,6 +923,51 @@ func (h *FaultHandler) startNetworkPacketLossFault(ctx context.Context, taskMeta
 	return nil
 }
 
+// stopNetworkPacketLossFault invokes the linux TC utility tool to stop the network-packet-loss fault.
+func (h *FaultHandler) stopNetworkPacketLossFault(ctx context.Context, taskMetadata *state.TaskResponse, request types.NetworkPacketLossRequest) error {
```

Comment:
Q: Are we using `request` anywhere?

Reply:
Thanks for the catch! Removing this in the next revision